src/components/test/context.jsx

```diff
@@ -24,6 +25,19 @@ class TestContext extends Component {
     ])
   };
 
+  renderVideo = (ctx, title) => {
+    const isUrl = urlRegEx.test(ctx);
+    const hasProtocol = protocolRegEx.test(ctx);
+    const linkUrl = (isUrl && !hasProtocol) ? `http://${ctx}` : ctx;
+
+    return (
+      <video controls src={ linkUrl } className={ cx('video') }>
+        <track kind='captions' />
```

Comment:
The accessibility checker flags video tags without captions even though there is caption source file. I didn't know what would be better - disabling the lint rule or fooling it. Either way there are no captions.